cloud_edition/flexibility_mode/docs/crontasks.rst

```diff
@@ -122,18 +124,20 @@ The default crontab at the moment on our Flexibility environments is the followi
-| pimee:franklin-insights:fetch-products --env=prod              | \*/30 \* \* \* \* | Every 30 minutes                          |
-+----------------------------------------------------------------+-------------------+-------------------------------------------+
-| akeneo:reference-entity:refresh-records --all --env=prod       | 0 23 \* \* \*     | At 11:00 PM                               |
-+----------------------------------------------------------------+-------------------+-------------------------------------------+
-| pimee:data-quality-insights:schedule-periodic-tasks --env=prod | 15 0 \* \* \*     | At 00:15 AM                               |
-+----------------------------------------------------------------+-------------------+-------------------------------------------+
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| Symfony console command                                                           | Crontab frequency   | Human frequency          |
++===================================================================================+=====================+==========================+
+| akeneo:rule:run --env=prod                                                        | 0 5 \* \* \*        | At 05:00 AM              |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:project:notify-before-due-date --env=prod                                   | 20 0 \* \* \*       | At 12:20 AM              |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:project:recalculate --env=prod                                              | 0 2 \* \* \*        | At 02:00 AM              |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:franklin-insights:fetch-products --env=prod                                 | \*/30 \* \* \* \*   | Every 30 minutes         |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| akeneo:reference-entity:refresh-records --all --env=prod                          | 0 23 \* \* \*       | At 11:00 PM              |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:data-quality-insights:schedule-periodic-tasks --env=prod                    | 15 0 \* \* \*       | At 00:15 AM              |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:franklin-insights:quality-highlights:push-structure-and-products --env=prod | 15 0,12 \* \* \* | At 12:15 AM and 12:15 PM    |
```

Comment:
the table will not be correctly displayed missing spaces in crontab frequency to correctly align the `|`